docs/examples/KModal/KModal.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+
+  <div>
+    <KButton @click="openModal">Click Me</KButton>
+    <KModal
+      v-if="showModal"
+      :appendToOverlay="true"
+      title="Dialog Box"
```

Comment:
```suggestion
      title="Dialog box"
```

Aligns case with writing guidelines.